packages/common/src/api/tan-query/queryKeys.ts

```diff
@@ -9,6 +9,8 @@ export const QUERY_KEYS = {
   users: 'users',
   userByHandle: 'userByHandle',
   userTracksByHandle: 'userTracksByHandle',
+  userPlaylists: 'userPlaylists',
+  userAlbums: 'userAlbums',
```

Comment:
thoughts on `playlistsByUser` to match `userByHandle`, etc?

Reply:
yeah no strong feelings, @DejayJD curious what you think?